cadquery/cq.py

```diff
@@ -3041,46 +3036,40 @@ def extrude(
         *  if combine is true, the value is combined with the context solid if it exists,
            and the resulting solid becomes the new context solid.
         """
-        if combine == "cut":
-            return self.cutBlind(until, clean, taper)
-        else:
-            # Handle `until` multiple values
-            if isinstance(until, str) and until in ("next", "last") and combine:
-                if until == "next":
-                    faceIndex = 0
-                elif until == "last":
-                    faceIndex = -1
 
-                r = self._extrude(None, both=both, taper=taper, upToFace=faceIndex)
+        # Handle `until` multiple values
+        if isinstance(until, str) and until in ("next", "last") and combine:
```

Comment:
If combine is  "cut" the conditional test will be True but the executed code won't be what's expected. 
That's why I added an explicit condition to check if `combine="cut"` and if this True we call the `cutBlind` method.
I think we should keep my first test

Reply:
Thanks, missed that. I think it is now fixed.